src/udev/udevadm-hwdb.c

```diff
@@ -677,7 +688,14 @@ static int adm_hwdb(struct udev *udev, int argc, char *argv[]) {
                 }
                 trie->nodes_count++;
 
-                err = conf_files_list_strv(&files, ".hwdb", root, conf_file_dirs);
+                char *conf_file_path = list_conf_file_path ();
```

Comment:
Why not annotate this as `_cleanup_free_` and get rid of the `free` below?

Reply:
That’s true, _cleanup_free_ is the way to go